From 2f8a57f66a94210d34dd0361659a8297cf3d813f Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 8 Jan 2016 07:29:24 +0000
Subject: [PATCH 29/32] trace_shed: Fix build with musl

Errors on musl are

trace_sched.c:425:16: error: invalid operands to binary & (have
'thread_sched_t * {aka struct <anonymous> *}' and 'int')
      thrd_ndx, WEXITSTATUS(status));

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 testcases/kernel/sched/tool/trace_sched.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testcases/kernel/sched/tool/trace_sched.c b/testcases/kernel/sched/tool/trace_sched.c
index 7815686..30dca5f 100644
--- a/testcases/kernel/sched/tool/trace_sched.c
+++ b/testcases/kernel/sched/tool/trace_sched.c
@@ -422,7 +422,7 @@ int main(int argc,		/* number of input parameters.                        */
 			if (status == (thread_sched_t *) - 1) {
 				fprintf(stderr,
 					"thread [%d] - process exited with errors %d\n",
-					thrd_ndx, WEXITSTATUS(status));
+					thrd_ndx, WEXITSTATUS((int)status));
 				exit(-1);
 			} else {
 				exp_prio[thrd_ndx] = status->exp_prio;
-- 
2.7.0

